Corona-Warn-App/src/main/res/layout/trace_location_organizer_qr_code_detail_fragment.xml

```diff
@@ -167,6 +168,16 @@
 
     </androidx.coordinatorlayout.widget.CoordinatorLayout>
 
+    <View
+        android:id="@+id/shadow_view"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@drawable/shadow"
+        app:layout_constraintBottom_toTopOf="@id/qr_code_print_button"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/coordinator_layout" />
+
```

Comment:
New design requirement?

Reply:
No specs 🤓 for this though

Reply:
Aligned with UX team , Please refer to the lasted video demo ☝️